ansible/roles/common/tasks/main.yml

```diff
@@ -63,3 +73,14 @@
     dest: /etc/nginx/nginx.conf
     regexp: 'server_names_hash_bucket_size'
     line: '	server_names_hash_bucket_size 64;'
+
+- name: nginx default site conf
+  template: src=nginx/default
+            dest=/etc/nginx/sites-enabled/default
+
+- name: remove default nginx site html page
```

Comment:
Yeah, that's right. Re-creating it is a waste. The original default page is named `index.nginx-debian.html`, so maybe the right fix would be to always remove that file if it's there.
Or to avoid removing `index.html`, as we're creating that one from the template.